src/style.css

```diff
@@ -0,0 +1,32 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row-reverse;
+  width: fit-content;
```

Comment:
please remove it, this styles is useless. also after removing you should see that the starts are placed in reversed order, so you need to fix it in `nth-of-type` pseudo class below
```suggestion
```